File: inflearn_basic_DB/board_dsl/src/main/java/com/dooho/board/api/comment/CommentDto.java
package com.dooho.board.api.comment;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CommentDto {
            private Integer commentId;
    private Integer boardNumber;
    private String userEmail;
    private String commentUserProfile;
    private String commentUserNickname;
    private LocalDate commentWriteDate;
    private String commentContent;
}

File: board_Kotlin/src/main/java/com/dooho/board/api/comment/CommentDto.kt
package com.dooho.board.api.comment

import java.time.LocalDate

data class CommentDto(

    val commentId: Int? = null,
    val boardNumber: Int? = null,
    val userEmail: String? = null,
    val commentUserProfile: String? = null,
    val commentUserNickname: String? = null,
    val commentWriteDate: LocalDate? = null,
    val commentContent: String? = null,

)